src/views/auth/Signin.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+<sbc-signin
+    @sync-user-profile-ready="redirectOnLogin()"
+  ></sbc-signin>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+
+// Components
+import SbcSignin from 'sbc-common-components/src/components/SbcSignin.vue'
+
+@Component({
+  components: {
+    SbcSignin
+  }
+})
+export default class SigninView extends Vue {
+  private async mounted () {
+    console.log('[SigninView] -- Mounted')
+  }
+  private async redirectOnLogin () {
+    console.log('[SigninView] -- kebab-style-found')
+    this.$router.push('/draft-company')
```

Comment:
Is this a valid route

Reply:
Corrected in the latest commit, thanks !